files/image_config/monit/monit_multiprocesses.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
```

Comment:
I think this file should be broken into two separate files, one for teamd and one for dhcp_relay. In the repo, the files should reside in the directories of their respective dockers.